docs/general-development/common-error-messages-in-sharepoint-workflow-development.md

```diff
@@ -9,11 +9,15 @@ localization_priority: Priority
 
 # Common error messages in SharePoint workflow development
 A listing of common error messages that you might encounter while developing SharePoint workflows and guidance for solving the underlying problem.
+
+> [!NOTE]
+> SharePoint 2010 workflows has been retired since August 1, 2020. If you’re using SharePoint 2010 workflows, we recommend migrating to Power Automate or other supported solutions. For more info, see [SharePoint 2010 workflow retirement](https://support.microsoft.com/office/sharepoint-2010-workflow-retirement-1ca3fff8-9985-410a-85aa-8120f626965f).

```

Comment:
@laschultz We should update this to reflect new tenants and existing tenants:

_SharePoint 2010 workflows have been retired since August 1, 2020, for new tenants and will be removed from existing tenants on November 1st, 2020._

Can you review the text and update it accordingly? 

For the link to the support article, I see you have referenced the actual link - should we use a fwlink instead?



Thanks,
Chaks

Reply:
@chakkaradeep OK will do. Question though, should I leave the sentence in about Power Automate?

Reply:
Yep.

Reply:
@chakkaradeep OK, the update note will read "SharePoint 2010 workflows have been retired since August 1, 2020, for new tenants and will be removed from existing tenants on November 1st, 2020. If you’re using SharePoint 2010 workflows, we recommend migrating to Power Automate or other supported solutions. For more info, see [SharePoint 2010 workflow retirement]
(https://support.microsoft.com/office/sharepoint-2010-workflow-retirement-1ca3fff8-9985-410a-85aa-8120f626965f)."